composer.json

```diff
@@ -26,14 +26,14 @@
         "nelmio/security-bundle": "~1.4",
         "doctrine/doctrine-migrations-bundle": "~1.0.0",
         "doctrine/migrations": "~1.0.0|1.0.0@alpha",
-        "mopa/bootstrap-bundle": "3.0.0-RC2",
+        "mopa/bootstrap-bundle": "3.0.0",
```

Comment:
Let's say I feel it's risky. I'd only upgrade after manual checking. Although it sounds like a good idea, I have no knowledge of a SURF-initiated push towards Symfony 3. The next LTS is released in November 2017.